android/app/src/main/cpp/vncconn.c

```diff
@@ -369,6 +369,14 @@ JNIEXPORT jboolean JNICALL Java_com_coboltforge_dontmind_multivnc_VNCConn_rfbIni
         cl->serverHost = strdup(cHost);
         (*env)->ReleaseStringUTFChars(env, host, cHost);
     }
+    if (0 == strcmp("localhost", cl->serverHost) || 0 == strcmp("127.0.0.1", cl->serverHost))
```

Comment:
* style: please put the 0 as second place in the comparison
* logic: please also handle the IPv6 case.